package.json

```diff
@@ -28,6 +28,7 @@
         "start": "npm run clean && tsc --watch"
     },
     "dependencies": {
+        "chalk": "^4.1.2",
```

Comment:
I added chalk for highlighting warnings and errors. I had to use v4 because v5 is ESM only. chalk@4 was already installed because it's also a dependency of other dependencies

https://www.npmjs.com/package/chalk